src/app/(auth)/login/page.tsx

```diff
@@ -51,7 +51,7 @@ export default function Login() {
   return (
     <>
       <SpacerDiv $gap={0.625}>
-        <H1>Log In</H1>
+        <AuthHeading>Log In</AuthHeading>
         {errorMessage !== '' && <P $color={COLORS.redMid}>{errorMessage}</P>}
       </SpacerDiv>
       <SpacerDiv $gap={0.8125}>
```

Comment:
I don't think this SpacerDiv is necessary here either, since it is directly nesting another SpacerDiv. 
Additionally, the default gap value of SpacerDiv is 19.2px currently, but I think we want it to be 20px.